scheduler/conf.c

```diff
@@ -417,15 +417,6 @@ cupsdDefaultAuthType(void)
     return (default_auth_type);
 
 #ifdef HAVE_GSSAPI
-#  ifdef __APPLE__
- /*
-  * If the weak-linked GSSAPI/Kerberos library is not present, don't try
-  * to use it...
-  */
-
```

Comment:
Required code.

Reply:
Kerberos is always present on macOS though and so are the other libraries. Maybe I should do another PR where this is a compile time check? Or does it need to be a runtime one

Reply:
@AtariDreams Kerberos is not part of the macOS installer or when you boot to the recovery console. Since libcups is linked to the ApplicationServices framework, a hard dependency will cause these environments to crash hard, effectively bricking a system...

This run-time check was added years ago for this reason...

Reply:
Of course it was added for a reason. It’s just that, as to my knowledge, the CUPS running in the recovery console will be apple’s fork, not this version. I’ll revert the changes anyway.